src/context/userStore.ts

```diff
@@ -403,6 +409,30 @@ export const useUserStore = create<UserState>()(
             showError('Transfer', e)
           }
         },
+        claimTokens: async () => {
+          const user_address = get().address
+          const handleOp = get().handleOp
+          const showError = useModalStore.getState().showError
+          const step = useModalStore.getState().step
+
+          step('Claiming Teia DAO tokens', 'Claiming Teia DAO tokens', true)
+
+          // Download the distribution file mapping file
+          const mapping = await downloadFileFromIpfs(DISTRIBUTION_MAPPING_IPFS_PATH)
+
+          // Download the Merkle data
+          const merkleDataPath = MERKLE_DATA_IPFS_PATHS[mapping[user_address]]
+          const merkleData = await downloadFileFromIpfs(merkleDataPath)
+          console.log(merkleData)
+          try {
+            const dropContract = await Tezos.wallet.at(DAO_TOKEN_CLAIM_CONTRACT)
+            const batch = dropContract.methods.claim(merkleData.proof, merkleData.leafDataPacked)
```

Comment:
This fails at the moment because the current claim contract was deployed to the testnet and not to the mainnet